frontend/src/locales/fr.json

```diff
@@ -145,13 +145,13 @@
     },
     "dashboard": {
       "selectFilter": {
-        "clear": "Clair"
+        "clear": "Vider"
       }
     },
     "dialog": {
       "dialogEntityDelete": {
-        "title": "Vraiment effacer?",
-        "warningText": "Voulez-vous vraiment le supprimer ?"
+        "title": "Confirmation de suppression",
+        "warningText": "Veux-tu vraiment supprimer ce bloc ?"
```

Comment:
This message is a general message used in multiple deletion dialogs. I think it's used for deleting, activities, pieces of an activity, periods, entire camps, and maybe more. This message should remain general for now, until the individual dialogs define more specific messages.

Reply:
I replace "ce bloc" by "cet élément". It's more generic. But because in French we don't have the neutral pronoun, we need to use a workaround...